bin/templates/cordova/lib/build.js

```diff
@@ -82,7 +82,9 @@ function parseOpts (options, resolvedTarget, projectRoot) {
         if (config.android && config.android[ret.buildType]) {
             var androidInfo = config.android[ret.buildType];
             if (androidInfo.keystore && !packageArgs.keystore) {
-                if (androidInfo.keystore.substr(0, 1) === '~') {
+                if (androidInfo.keystore.substr(0, 1) === '~' 
+                    || androidInfo.keystore.substr(0, 5) === '$HOME' 
+                    || androidInfo.keystore.substr(0, 10) === '%HOMEPATH%') {
                     androidInfo.keystore = process.env.HOME + androidInfo.keystore.substr(1);
```

Comment:
This line is now broken as `androidInfo.keystore.substr(1)` removes the first character from the keystore string - which only works for `~`

Reply:
I dont know how it can be broken?

Reply:
oh I see that, I should make an other check

Reply:
I fix that